packages/api-specification/interface/screen.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace ssf {
+  export abstract class Screen {
+    static getDisplays(): Promise<Array<Display>>;
+  }
+
+  /**
+   * Information about the users display. Note that the Browser API does not support multiple
+   * displays, so it assumes the display the browser is running in is the only display.
+   */
+  export interface Display {
+    /**
+     * Unique Id of the display.
+     */
+    id: string;
```

Comment:
I think it's worth adding - there's an open issue on that topic for electron: https://github.com/electron/electron/issues/1836